build.psm1

```diff
@@ -976,8 +993,8 @@ function Start-PSBootstrap {
             $newMachineEnvironmentPath = $machinePath
 
             $cmakePresent = precheck 'cmake' $null
-            $win10sdkBinPath = "${env:ProgramFiles(x86)}\Windows Kits\10\bin\x64"
-            $sdkPresent = Test-Path -Path $win10sdkBinPath -PathType Container
+            $win10sdkBinPath = Get-Win10SDKBinDir
```

Comment:
`$win10sdkBinPath` declared but not used in `Start-PSBootstrap`. Can you please remove it?